sources/install/package_ad.sh

```diff
@@ -118,13 +118,81 @@ function install_bloodhound() {
     colorecho "Installing BloodHound from sources"
     git -C /opt/tools/ clone --depth 1 https://github.com/BloodHoundAD/BloodHound/
     mv /opt/tools/BloodHound /opt/tools/BloodHound4
-    zsh -c "source ~/.zshrc && cd /opt/tools/BloodHound4 && nvm install 16.13.0 && nvm use 16.13.0 && npm install -g electron-packager && npm install && npm run build:linux"
+    zsh -c "source ~/.zshrc && cd /opt/tools/BloodHound4 && nvm install 16.13.0 && nvm use 16.13.0 && npm install -g electron-packager && npm install && npm run build:linux && nvm use default"
```

Comment:
why the `nvm use default` at the end?

Reply:
Because bh needs to build with node 16.13.0 and at the end of the installation, I switch to the default version of node (v20.8.0)